nvim/stdio_stream.lua

```diff
@@ -0,0 +1,39 @@
+local uv = require('luv')
+
+
+local StdioStream = {}
+StdioStream.__index = StdioStream
+
+function StdioStream.open()
```

Comment:
It's not required and not even valid. In this case we are already explicitly defining the function as the `open` field of `StdioStream`(something like a class method)